README.md

```diff
@@ -84,6 +84,8 @@ Command | Description
 --- | ---
 `rabbitmqctl list_queues` | List all queues
 `rabbitmqctl purge_queue home` | Remove all messages from `home` queue
+`rabbitmqctl status` | check if rabbitmq server is currently running  
+`rabbitmq-server`    | start rabbitMQ server  
```

Comment:
Could you clarify if you met an issue and in order to solve it you needed to run `rabbitmq-server`? The same for the command `redis-server`.

Assuming that rabitmq and redis should be run in Docker. Could you specify in the description you introduced that `server will be run in Docker container`

Reply:
Ok. I see.